Assets/multilingual font compression (#1322)
massgov/mayflower: packages/assets/static/fonts/readme.md

4. If not, copy the entire block for a script, paste it to the command line and enter
5. Run git status to test if the font files were updated

#### Troubleshooting

> - The most common error is that the font is not found which means the first line of the command doesnt match a file name in the directory you are in.

Reviewer: ```suggestion
- The most common error is that the font is not found which means the first line of the command doesn't match a file name in the directory you are in.
```

> - If the command does not workas one block, try running them one at a time (make sure to exlude the &&).

Reviewer: ```suggestion
- If the command does not work as one block, try running them one at a time (make sure to exclude the `&&`).
```

> - Don't forget to cd into the right direecctory for the script first.

Reviewer: ```suggestion
- Don't forget to cd into the right directory for the script first.
```
